tools/steam-shell.pike

```diff
@@ -458,25 +458,22 @@ int list(string what)
     write("Wrong usage\n");
     return 0;
   }
-  int flag=0;
   string toappend="";
   array(string) display = get_list(what);
-  string a="";
   if(sizeof(display)==0)
-    toappend = "There are no "+what+" in this room\n";
-  else
+    toappend = "There are no "+what+" in this room.\n";
+  else if (display[0]=="Invalid command")
+  {
+    write(display+"\n");
```

Comment:
applauncher.pike:183: Warning: Unused local variable status.
steam-shell.pike:467:Too few arguments to `+ (got 2).
steam-shell.pike:467:Expected: object | mixed.
steam-shell.pike:467:Function type:
steam-shell.pike:467:Got     : scope(0,function((0=multiset), (1=multiset) ... : 0 | 1) | function((0=mapping), (1=mapping) ... : 0 | 1) | function((0=array), (1=array) ... : 0 | 1) | function(int, int ... : int) | !function(!string ... : mixed) & function(string | int | float, string | int | float ... : string) | !function(!float ... : mixed) & function(int | float, int | float ... : float) | !function(!(object | mixed) ... : mixed) & function(mixed ... : mixed)).
Pike: Failed to compile script.

I got this error on running steam-shell.
Got fixed on changing like 467 to display[0].